CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## unreleased
 
+* Make `BrowserSwitchClient.assertCanPerformBrowserSwitch()` public
+
```

Comment:
```suggestion
* Make `BrowserSwitchClient.assertCanPerformBrowserSwitch()` public
```

Reply:
I'm not seeing any differences in the suggested change. Can you update the snippet?

Reply:
Oh I was trying to remove the newline after the entry - nitpick it's fine either way

Reply:
Ah I see. I just removed the newline 👍